### PR TITLE
Add dependencies, packages and update lockfileVersion

### LINKED ISSUE #5: Old lockfile when installing
Steps to replicate:

1. Run `npm install @nasa-jpl/stellar --save` as instructed in the README
2. Encounter `npm WARN old lockfile`:

```
npm WARN old lockfile                                                                                                                       
npm WARN old lockfile The package-lock.json file was created with an old version of npm,                                                    
npm WARN old lockfile so supplemental metadata must be fetched from the registry.                                                           
npm WARN old lockfile                                                                                                                       
npm WARN old lockfile This is a one-time fix-up, please be patient...                                                                       
npm WARN old lockfile 
```

Suggested fix -- commit the updated `package-lock.json` and `package.json` to the repository that is generated, post-install.

### PR DESCRIPTION
**Pull request by wbnns**
Resolves #5

### CODE REVIEW

**Comment by camargo** [comment]
Thanks for the contribution! Can you also bump the package.json version, and npm install so the package-lock.json version updates as well?

File: package.json [hunk 1]
```diff
@@ -18,5 +18,8 @@
     "stylelint": "^14.5.3",
     "stylelint-config-standard-scss": "^3.0.0",
     "stylelint-order": "^5.0.0"
+  },
+  "dependencies": {
+    "@nasa-jpl/stellar": "^1.0.5"
```
Comment (camargo) [inline package.json]: The package should not depend on itself

Comment (wbnns) [reply package.json]: Removed in [b39dd47](https://github.com/nasa-jpl/stellar/pull/6/commits/b39dd4759e133330fd1bba3ba4cdafa9faa30aa7).

**Comment by wbnns** [comment]
@camargo 

Thanks for the review! Removed the dependency (apologies, I think that may have been introduced when I ran `npm install` before).

Also pushed [0be2e8e](https://github.com/nasa-jpl/stellar/pull/6/commits/0be2e8ee03a9c73e86b542c875255603181fed99) to bump the version and re-ran `npm install` once more to update `package-lock.json` as requested.

If there's anything else you'd like to see here just let me know. 👍️

**Comment by camargo** [comment]
Will update this separately

**Pull request closed without merging**